### ISSUE TITLE
compat with yarn berry 

### ISSUE THREAD

**Issue by birkskyum** [nksaraf/vinxi#43]
using solid start with yarn berry result in module resolution issues with jiti, through vinxi:

- https://github.com/solidjs/solid-start/issues/1177#issuecomment-1869484191

repro:
- `yarn create solid`
- select `bare` (important step that is the `bare` template)
- `yarn set version berry` (to use yarn v4)
- `yarn`
- `yarn dev`

**Comment by nksaraf** [comment]
you have any idea what might be wrong? I havent use yarn (or berry) in a very long time.

**Comment by birkskyum** [comment]
yes, i believe @danieltroger managed to narrow it down quite a lot in here https://github.com/solidjs/solid-start/issues/1177#issuecomment-1869484191

**Comment by pi0** [comment]
Some notes here: https://github.com/unjs/jiti/issues/204#issuecomment-1894357666

TLDR: I think vinxi can help to resolve complexity of situation by preferring `type: "module"` + `"default"` condition.

**Comment by nksaraf** [comment]
Type module is already there I think. Default condition for what? All the exports ? Sure I'll do that
